build-tests/api-extractor-test-01/etc/api-extractor-test-01.api.md

```diff
@@ -49,11 +49,11 @@ export class ClassWithAccessModifiers {
 // @public (undocumented)
 export class ClassWithSymbols {
     // (undocumented)
-    readonly [unexportedCustomSymbol]: number;
+    [fullyExportedCustomSymbol](): void;
```

Comment:
The order changed here because the sort key is based on the *actual* property name (i.e. `[unexportedCustomSymbol]`) rather than the (early) bound symbol name of `__computed`.